contrib/pg_stat_statements/expected/pg_stat_statements.out

```diff
@@ -458,22 +484,57 @@ SELECT pg_stat_statements_reset(NULL);
-------------------------------------------------------------------------------+-------+------
- CREATE ROLE regress_stats_user1                                              |     1 |    0
- CREATE ROLE regress_stats_user2                                              |     1 |    0
- RESET ROLE                                                                   |     2 |    0
- SELECT $1 AS "ONE"                                                           |     1 |    1
- SELECT $1 AS "ONE"                                                           |     1 |    1
- SELECT $1+$2 AS "TWO"                                                        |     1 |    1
- SELECT $1+$2 AS "TWO"                                                        |     1 |    1
- SELECT pg_stat_statements_reset($1)                                          |     1 |    1
- SELECT pg_stat_statements_reset()                                            |     1 |    1
- SELECT query, calls, rows FROM pg_stat_statements ORDER BY query COLLATE "C" |     1 |   10
- SET ROLE regress_stats_user1                                                 |     1 |    0
- SET ROLE regress_stats_user2                                                 |     1 |    0
-(12 rows)
+SELECT count(*), query, sum(calls) calls, sum(rows) rows FROM pg_stat_statements GROUP BY query ORDER BY query COLLATE "C";
+ count |                                                           query                                                            | calls | rows 
+-------+----------------------------------------------------------------------------------------------------------------------------+-------+------
+     1 | CREATE ROLE regress_stats_user1                                                                                            |     1 |    0
+     3 | CREATE ROLE regress_stats_user1;                                                                                           |     3 |    0
+     1 | CREATE ROLE regress_stats_user2                                                                                            |     1 |    0
+     3 | CREATE ROLE regress_stats_user2;                                                                                           |     3 |    0
+     1 | RESET ROLE                                                                                                                 |     2 |    0
+     3 | RESET role                                                                                                                 |     6 |    0
+     2 | SELECT $1 AS "ONE"                                                                                                         |     2 |    2
+     2 | SELECT $1+$2 AS "TWO"                                                                                                      |     2 |    2
+     4 | SELECT -$2::int8 as gp_segment_id, * FROM pg_stat_statements_local($3)                                                    +|     5 |   29
```

Comment:
A very funny parameter `-$2`. I wonder, should it contain minus?

Reply:
Seems, it should. This minus actually refers to the T_TypeCast node, not the T_TypeConst